source/13-transitioning/03-transitioning-from-counter-r4-to-r5.rst

```diff
@@ -27,11 +27,11 @@ The transition from R4 to R5 meets the general requirements outlined in :numref:
 
    * - BR1
      - Full-text requests (at the book level)
-     - Unique_Title_Requests AND Data_Type=Book AND Section_Type=Book
+     - Unique_Title_Requests AND Data_Type=Book
 
    * - BR2
      - Full-text requests (at the chapter/section level)
-     - Total_Item_Requests AND Data_Type=Book AND Section_Type=Chapter|Section
+     - Total_Item_Requests AND Data_Type=Book
```

Comment:
Same question here as for section 1.2 and appendix B:,should this section really describe the changes from R4 to R5.1?